cmd/image/qcow2ova/prep/loop.go

```diff
@@ -104,6 +106,20 @@ func mount(opts, src, target string) error {
 func Umount(dir string) error {
 	exitcode, out, err := utils.RunCMD("umount", dir)
 	if exitcode != 0 {
+		if strings.Contains(err, "not mounted") {
+			klog.Infof("Ignoring 'not mounted' error for %s", dir)
+			return nil
+		}
+		if strings.Contains(err, "no mount point specified") {
+			klog.Infof("Ignoring 'no mount point specified' error for %s", dir)
+			return nil
+		}
+		if strings.Contains(err, "target is busy") {
+			exitcode, out, err = utils.RunCMD("umount", "-lf", dir)
```

Comment:
in what scenario we need this? lets see if we can somehow avoid the lazy mount.

Reply:
Whenever an active process is currently accessing the target, we get the target  is busy error.

When  we press Ctrl+c while preparing the image, a child process (executing setup.sh) is still accessing the mount point, when we kill the child process, the pvsadm process is also getting killed immediately which leads to dangling mount points. 
In order to handle this scenario, added retrying of unmount for 5 times to wait for the process to end and added lazy unmount after retry.